sql_generators/stable_views/__init__.py

```diff
@@ -23,6 +23,8 @@
 from bigquery_etl.dryrun import get_id_token
 from bigquery_etl.config import ConfigLoader
 
+BOT_GENERATED = 'LOWER(IFNULL(metadata.isp.name, '')) <> "browserstack" AS bot_generated,'
```

Comment:
Isn't `bot_generated = true` when the ISP name is *not* "browserstack"?  Is that not backward?